jina/optimizers/__init__.py

```diff
@@ -0,0 +1,105 @@
+from pathlib import Path
+from ruamel.yaml import YAML
+
+from jina.helper import colored
+from jina.logging import default_logger as logger
+from .parameters import load_optimization_parameters
+
+
+class EvaluationCallback:
+    def __init__(self, op_name=None):  # Todo: better name than op_name
```

Comment:
please add type hinting

Reply:
I will add typehint and docstring once we freeze the code